src/DynamicRupture/FrictionLaws/FrictionSolver.h

```diff
@@ -16,7 +16,11 @@ namespace seissol::dr::friction_law {
 class FrictionSolver {
   public:
   // Note: FrictionSolver must be trivially copyable. It is important for GPU offloading
-  explicit FrictionSolver(dr::DRParameters* userDrParameters) : drParameters(userDrParameters){};
+  explicit FrictionSolver(dr::DRParameters* userDrParameters) : drParameters(userDrParameters) {
+    real points[NUMBER_OF_SPACE_QUADRATURE_POINTS][2];
+    std::fill_n(spaceWeights, dr::misc::numPaddedPoints, static_cast<real>(0.0));
+    seissol::quadrature::TriangleQuadrature(points, spaceWeights, CONVERGENCE_ORDER + 1);
```

Comment:
[Also](seissol::quadrature::TriangleQuadrature) is also called in DynamicRupture.cpp. Is it needed?

Reply:
You're right. It is not needed. I removed it.

done